samcli/lib/init/templates/cookiecutter-aws-sam-hello-nodejs/{{cookiecutter.project_name}}/hello-world/package.json

```diff
@@ -7,7 +7,7 @@
   "author": "SAM CLI",
   "license": "MIT",
   "dependencies": {
-    "axios": "^0.21.1"
+    "axios": "^1.6.0"
```

Comment:
Can we also bump all of the occurrences in the init [templates](https://github.com/aws/aws-sam-cli-app-templates/blob/master/nodejs14.x/hello/%7B%7Bcookiecutter.project_name%7D%7D/hello-world/package.json#L10) for consistency?